webpack.config.js

```diff
@@ -52,6 +52,7 @@ module.exports = {
   plugins: [
     new webpack.EnvironmentPlugin([]),
     new webpack.EnvironmentPlugin(['STRIPE_API_KEY']),
+    new webpack.EnvironmentPlugin(['SENDGRID_API_KEY']),
```

Comment:
You can leave out this line in your webpack config, since the front end should not receive the sendgrid key. some API keys are ok to bundle in your front end code, but the sendgrid key is _definitely_ not one of them